format-delimited/src/main/java/io/cdap/plugin/format/delimited/input/PathTrackingDelimitedInputFormat.java

```diff
@@ -93,6 +93,7 @@ public NullWritable getCurrentKey() {
             int numDataFields = 0;
             splitsIterator = getSplitsIterator(enableQuotesValue, delimitedString, delimiter);
             while (splitsIterator.hasNext()) {
+              splitsIterator.next();
```

Comment:
This is interesting, if we don't have this change, the logic will be here forever. Does this mean this functionality is never used by any customer or we don't have any unit test for it?

Reply:
I think we don't have any unit tests. As I've hit this issue while testing.